deployment/configutils/CMakeLists.txt

```diff
@@ -37,6 +37,7 @@ include_directories (
          ${HPCC_SOURCE_DIR}/deployment/deploy
          ${CMAKE_BINARY_DIR}/oss
          ${CMAKE_BINARY_DIR}
+         ${HPCC_SOURCE_DIR}/system/security/shared
```

Comment:
For consistency I followed whatever pattern was established in each CMakeList.txt file.  I do agree  it's time for all of them to be normalized though